ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/LedgerOffsetValidator.scala

```diff
@@ -32,12 +32,12 @@ object LedgerOffsetValidator {
       ledgerOffset: LedgerOffset,
       fieldName: String,
   ): Either[StatusRuntimeException, domain.LedgerOffset] = {
-    ledgerOffset match {
-      case LedgerOffset(LedgerOffset.Value.Absolute(value)) =>
+    ledgerOffset.value match {
```

Comment:
Is this just cleanup or does the old code no longer work?

Reply:
Cleanup; I reverted the other changes in this file (adding `_` to discard unknown fields everywhere) but thought I'd leave this one alone, because I find it easier to read.

Reply:
fine with me, just wanted to check